src/api/consumer/lib/untrack.js

```diff
@@ -0,0 +1,40 @@
+/** @flow */
+import includes from "lodash.includes";
+import R from "ramda";
+import BitMap from "../../../consumer/bit-map";
+import {loadConsumer, Consumer} from "../../../consumer";
+import ComponentsList from "../../../consumer/component/components-list";
+
+
+export default async function untrack(componentIds: string[]): Promise<Object> {
+
+  const untrackedComponents = [];
+  const consumer: Consumer = await loadConsumer();
+  const bitMap = await BitMap.load(consumer.getPath());
+  const componentsList = new ComponentsList(consumer);
+  const newComponents = await componentsList.listNewComponents(true);
+  const stagedComponents = await componentsList.listExportPendingComponents();
+  const modifiedComponent = await componentsList.listModifiedComponents(true);
```

Comment:
I think the logic here should be simpler
If component ids is empty just run over the newComps and remove them.
If it's not empty, run over them, and for each one:
check if is missing, if not, check if it's in the new list if yes remove, if not add to the unRemovabale.